docs/installation/index.md

```diff
@@ -18,19 +18,36 @@ The Autoware Foundation provides no support on other platforms than those listed
 
 Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
 
-## Installation steps of Autoware
+## Installation of Autoware
 
-See [here](autoware) for how to install Autoware.
+There are two ways to set up Autoware. Choose one according to your preference.
 
-## Installation steps of tools for users
+### 1. Docker installation
+
+Docker can ensure that all developers in a project have a common, consistent development environment.
+It is recommended for beginners, light users, people who do not use Ubuntu.
+
+See [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/installation/autoware/docker-installation.md) for the detailed steps.  
+Refer to [autowarefoundation/autoware/docker/README.md](https://github.com/autowarefoundation/autoware/blob/main/docker/README.md) as well.
+
+### 2. Source installation
+
+Source installation is for the cases where more granular control of the installation environment is needed.
+It is recommended for skilled users or people who want to customize the environment.  
+Note that some problems may occur depending on your local environment.
+
+See [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/installation/autoware/source-installation.md) for the detailed steps.  
+Refer to the [autowarefoundation/autoware/README.md](https://github.com/autowarefoundation/autoware/blob/main/README.md) as well.
+
+## Installation of related tools
 
 Some other tools are required depending on the evaluation you want to do.
 For example, if you run E2E simulation, you need to install a simulator for that.
```

Comment:
It may not be obvious to someone what `E2E` means and it is only referred to once on this page, so it is better to write out that in full.

```suggestion
For example, to run an end-to-end simulation you need to install an appropriate simulator.
```